src/contexts/MintBurnFormContext/MintBurnFormContext.tsx

```diff
@@ -490,6 +494,27 @@ export const MintBurnFormProvider: React.FunctionComponent<MintBurnFormProviderP
       }
     }
 
+    // Check that the input margin is >= minimum required margin if removing margin
+    if (
```

Comment:
whats up with the nesting of ifs... you can merge 2 of the ifs into 1. Also declare a const variable telling us what exactly this logic does...

If you name it well you can remove the comments